distribution/zip/jballerina-tools/build.gradle

```diff
@@ -507,17 +507,17 @@ artifacts {
     unzipped file: file("$buildDir/extracted-distributions/${project.name}-${project.version}"), builtBy: createDistribution
 }
 
-// This section makes sure that checkstyle/spotbug runs when dist is build.
-project.afterEvaluate {
-    configurations.dist.dependencies
-        .findAll { it instanceof ProjectDependency }
-        .each {
-            it.dependencyProject.afterEvaluate {
-                def dependencyCheck = it.tasks.find { it.name == 'check' }
-                check.dependsOn dependencyCheck
-            }
-        }
-}
+////// This section makes sure that checkstyle/spotbug runs when dist is build.
```

Comment:
Can we remove this?

Reply:
I'll revisit the commented out lines and will remove if they are not necessary

Reply:
Evaluating a task second time has been removed from the Gradle version 7 and we need to make sure spotbugs and checkstyle checks run on the dist build.